docs/Signing.md

```diff
@@ -6,20 +6,23 @@ and information about the strongname or certificate that should be used.
 
 ## Format
 
-The signing request manifest supports three element types. A minimal example looks like this. See [Elements](#Elements) below for details
+The signing request manifest supports multiple element types. Each element represents a specific kind of action to take on a file. A minimal example looks like this. See [Elements](#Elements) below for details
 
 ```xml
 <SigningRequest>
   <File Path="MyAssembly.dll" Certificate="MyCert" StrongName="MyStrongName" />
   <File Path="build/Another.dll" Certificate="MyCert" />
-  <Container Path="MyLib.1.0.0.nupkg" Type="nupkg" Certificate="NuGetCert">
+  <Nupkg Path="MyLib.1.0.0.nupkg" Certificate="NuGetCert">
     <File Path="lib/netstandard2.0/MyLib.dll" Certificate="MyCert" />
-  </Container>
-  <Container Path="MyVSTool.vsix" Type="vsix" Certificate="VsixCert">
+  </Nupkg>
+  <Vsix Path="MyVSTool.vsix" Certificate="VsixCert">
     <File Path="MyVSTool.dll" Certificate="MyCert" />
```

Comment:
Just so I'm clear, this is the dll inside the vsix right?

Reply:
Yes.